packages/office-ui-fabric-react/package.json

```diff
@@ -19,7 +19,7 @@
     "build": "node ../../scripts/build.js",
     "clean": "node ../../scripts/clean.js",
     "code-style": "node ../../scripts/code-style.js",
-    "start": "../../apps/fabric-website-resources && npm start",
+    "start": "cd ../../apps/fabric-website-resources && npm start",
```

Comment:
@kenotron removed this `cd` in #5920 so I'd double check with him prior to merge as it seemed intentional (unsure why myself)

Reply:
`npm start` doesn't work in PowerShell without it. Looks like we might need a middle-man script.

Reply:
nah you're good to merge this it was a type (the removal) see Ken's comment below